_pages/user-experience/failure-proof.md

```diff
@@ -41,13 +41,6 @@ The Failure to Proof URL is shown to users throughout the identity proofing proc
   - Users who enter valid information, but are not able to pass Login.gov’s proofing process because we cannot verify their information.
   - Users who have run out of attempts to verify their information.
 
-<figure>
-  <img src="{{ site.baseurl }}/assets/img/missing-docs.png"
```

Comment:
Can we remove the now-unused `missing-docs.png` file as well?

Reply:
Yes! Thank you for pointing that out.